### PR TITLE
Set environment

### PR DESCRIPTION
**Pull request by hundt-corbalt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +24,13 @@ Here's an example of how to use this action in your workflow:
 
 ```yaml
 - name: Run Command And Report DSO Event
-  uses: Enterprise-CMCS/mac-fc-report-dso-event@{ref}
+  uses: Enterprise-CMCS/mac-fc-report-dso-event@0.2.2
```
Comment (ben-harvey) [inline README.md:27]: I think people often copy-paste these examples, and when `0.2.2` is outdated they will copy-paste that :) That's why I put `{ref}` so it wouldn't work when copy-pasted and they would need to think about it some

**Review by ben-harvey — APPROVED** [review]
Looks good, just one nit in the README but I don't need another look

File: README.md (hunk 1 above)
Comment (hundt-corbalt) [reply README.md:27]: My thinking was:
- they might just put `*` and we don't want to encourage that (we want the default to be that they will pin to the latest released version at the time they set it up)
- we already have to make a change updating references (in action.yml) whenever we bump the version number, so changing it in one more place doesn't seem harder to me.

Comment (ben-harvey) [reply README.md:27]: Sure, makes sense, as long as we remember to do it :)

**Pull request merged**